### DISCUSSION TITLE
Missing API Key in email confirmation - Help?

### DISCUSSION THREAD

**Discussion by tommysafar** [open-meteo/open-meteo#875] [Q&A]
Hi everyone, I just subscribed to the API Standard on Open Meteo and expected to receive my API key via email but I have only received a confirmation of payment and cannot seem to find any reference to my API key on the email or on the Customer Portal that seems to be only made for managing the payment subscription. 

Does anyone know where I could recover my API key? Help would be much appreciated

Thanks in advance, 
Tommy

**Comment by patrick-zippenfenig** [comment]
Hi,
sorry for the delay. There was a temporary issue with email delivery. You should now have received your API. Please reach out to info@open-meteo.com in case you did not receive it.
Thanks
Patrick
